libafl/src/corpus/testcase.rs

```diff
@@ -65,6 +65,8 @@ where
     parent_id: Option<CorpusId>,
     /// If the testcase is "disabled"
     disabled: bool,
+    /// has found crash (or timeout) or not
+    found_objectives: bool,
```

Comment:
Like, let's make this a counter, and if noone uses it, fine, but if they do, even better